petclinic.yaml

```diff
@@ -11,16 +11,21 @@ tools:
     type: dockerImage
     image: maven:3.5.4-jdk-8
     entryPoint: sleep infinity
-    persistentVolumes:
+    volumes:
       - name: maven-repo
-        containerPath: /root/.m2
-    services:
+        mountPath: /root/.m2
+      - name: projects
+        mountPath: /projects
+    endpoints:
       - name: spring-boot
-        port: 8080
-    mountProjectsSources: true
+        public: true
+        targetPort: 8080
```

Comment:
why targetPort instead of port ? (I mean, if I have to write the file I prefer to use short names)

Reply:
Agree with you. Good question. But probably this question is more to @garagatyi @skabashnyuk @l0rd since I just copied `che-plugin.yaml` format.
We could have different formats for Devfile and `che-plugin.yaml` but I prefer to use the same terms not to confuse users each time when it switch between writing workspace configuration and plugin configuration.
So, we can consider updating the model of `che-plugin.yaml` before Che 7 Beta release.